DESCRIPTION

```diff
@@ -0,0 +1,47 @@
+Package: admiralonco
+Type: Package
+Title: Oncology extension package for ADaM in R Asset Library 
+Version: 0.0.0
+Authors@R: c(
+    person("Stefan", "Bundfuss", email = "stefan.bundfuss@roche.com", role = c("aut", "cre")),
+    person("Ross", "Farrugia", role = "aut"),
+    person("Amit", "Jain", role = "aut"),
+    person("Liming (Catherine)", "Jin", role = "aut"),
```

Comment:
Add other authors as: 
Andrew Smith
Cloris Xue 
Caihong Xia 
Karl Kennedy
Konstantina Koukourikou 
Mary Dee
Neharika Sharma 
Samia Kabi
Stephen Gormley
Annie Yang 

And contributors as: 
Gillian Yin
Aiming Yang
Gopi Vegesna

Reply:
Added